### PR TITLE
Task: site template; about and projects section

### PR DESCRIPTION
**Pull request by katestarks**
Work to be reviewed:
Site template
About me
Projects

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,2 +1,93 @@
+<!DOCTYPE html>
 <html>
+<head>
+
```
Comment (HazDuck) [inline index.html]: remove line?

File: index.html [hunk 2]
```diff
@@ -1,2 +1,93 @@
...
+    <title>Kate Starks</title>
+    <link rel='stylesheet' type='text/css' href='css/normalize.css'/>
+    <link rel='stylesheet' type='text/css' href='css/styles.css'/>
+    <link href="https://fonts.googleapis.com/css?family=Quicksand:300,500" rel="stylesheet">
+
```
Comment (HazDuck) [inline index.html]: remove line?

File: index.html [hunk 3]
```diff
@@ -1,2 +1,93 @@
...
+            </ul>
+        </div>
+    </section>
+    <section id="projects" class="projects">
+        <h1>Example Projects</h1>
+        <div class="projectItems project1">
+            <div class="cover">
+                <h1>Responsive web page</h1>
+                <a href=""><h1>view here</h1></a>
+                <a href="https://github.com/katestarks/PilotShop"><img src="IMG/GithubLlogo.png"></a>
```
Comment (HazDuck) [inline index.html:50]: could add an alt="" for imgs (dont think its required but is best practice)

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+	height: 10%;
+	background-color: #02C7A6;
+	position: absolute;
+	bottom: 0;
+}
+
+.about {
+	/*padding: 10% 5% 7% 5%;*/
+	overflow: auto;
+	/*background-color: #F5313E;*/
```
Comment (HazDuck) [inline css/styles.css]: if not required remove comments

File: css/styles.css [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+}
+
+.heroArrow {
+	height: 100%;
+	margin: 0 auto;
+}
+
+nav{
+	position: sticky;
+	position :-webkit-sticky;
```
Comment (HazDuck) [inline css/styles.css:45]: does this def do anything? i couldnt work out what it does

File: css/styles.css [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+	top: 0;
```
Comment (HazDuck) [inline css/styles.css:46]: dont think it does anything

File: css/styles.css [hunk 7]
```diff
@@ -0,0 +1,272 @@
...
+	background-color: #FFF;
```
Comment (HazDuck) [inline css/styles.css:47]: dont think it does anything

File: css/styles.css [hunk 8]
```diff
@@ -0,0 +1,272 @@
...
+	overflow: auto;
+	z-index: 10;
```
Comment (HazDuck) [inline css/styles.css:49]: dont think it does anything

File: css/styles.css [hunk 9]
```diff
@@ -0,0 +1,272 @@
...
+	text-decoration: none;
+	color: #02C7A6;
+}
+
+a:hover {
+	color: #FAC002;
+}
+
+nav li {
+	list-style-type: none;
```
Comment (HazDuck) [inline css/styles.css]: dont think it does anything

File: css/styles.css [hunk 10]
```diff
@@ -0,0 +1,272 @@
...
+	display: inline;
+	font-size: 2rem;
+	padding: 20px;
+}
+
+.navColor {
+	width: 100%;
+	height: 10%;
```
Comment (HazDuck) [inline css/styles.css:75]: use px rather than %?

File: css/styles.css [hunk 11]
```diff
@@ -0,0 +1,272 @@
...
+	background-color: #02C7A6;
+	position: absolute;
+	bottom: 0;
+}
+
+.about {
+	/*padding: 10% 5% 7% 5%;*/
+	overflow: auto;
```
Comment (HazDuck) [inline css/styles.css:82]: dont think this does anything

File: css/styles.css [hunk 12]
```diff
@@ -0,0 +1,272 @@
...
+
+.contentQuote {
+	font-size: 3rem;
+	font-weight: bold;
+}
+
+.projects {
+	padding: 10% 5% 7% 5%;
+	background-color: #FAC002;
+	width: 100%;
```
Comment (HazDuck) [inline css/styles.css]: dont think this does anything

**Review by HazDuck — COMMENTED** [review]
Looks goof- main thing is unneeded css that i've tried to identify. looked online to see if its ok to use a % for padding and most seemed to say its OK.

File: index.html (hunk 1 above)
Comment (katestarks) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (katestarks) [reply index.html]: Done

File: index.html (hunk 3 above)
Comment (katestarks) [reply index.html:50]: Awaiting technical solution

Comment (katestarks) [reply index.html:50]: Not adding alt tag to background images - not needed

File: css/styles.css (hunk 4 above)
Comment (katestarks) [reply css/styles.css]: Done

File: css/styles.css (hunk 5 above)
Comment (katestarks) [reply css/styles.css:45]: Fixes it for Safari

File: css/styles.css (hunk 7 above)
Comment (katestarks) [reply css/styles.css:47]: Whole lot together fixes the nav bar when it gets to the top of the page

File: css/styles.css (hunk 8 above)
Comment (katestarks) [reply css/styles.css:49]: As above

File: css/styles.css (hunk 9 above)
Comment (katestarks) [reply css/styles.css]: Done

File: css/styles.css (hunk 10 above)
Comment (katestarks) [reply css/styles.css:75]: Prefer % for resizing

File: css/styles.css (hunk 11 above)
Comment (katestarks) [reply css/styles.css:82]: Done

File: css/styles.css (hunk 12 above)
Comment (katestarks) [reply css/styles.css]: Done

**Review by HazDuck — COMMENTED** [review]
Tidy.

**Pull request merged**